src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -32,6 +32,8 @@
 
     <BuildArgs Condition="'$(PgoInstrument)' == 'true'">$(BuildArgs) /p:PgoInstrument=true</BuildArgs>
     <BuildArgs Condition="'$(EnablePoison)' == 'true'">$(BuildArgs) /p:DISABLE_CROSSGEN=true</BuildArgs>
+
+    <BuildArgs Condition="'$(DotNetBuildBuildWorkloads)' == 'true'">$(BuildArgs) /p:BuildWorkloads=true</BuildArgs>
```

Comment:
Not blocking at all but why do we need this indirection? Can't we just condition on `DotNetBuildBuildWorkloads` in sdk?

Reply:
no because that property is set at the orchestrator level